cmd/retry-storage.go

```diff
@@ -264,11 +277,10 @@ func (f *retryStorage) reInitUponDiskNotFound(err error) bool {
 // attempts three times before giving up.
 func (f *retryStorage) reInit(e error) (err error) {
 
-	// Only after initialization and minimum of one interval
-	// has passed (to prevent marking a node as offline right
-	// after initialization), check whether node has gone offline
-	if f.maxRetryAttempts == globalStorageRetryThreshold &&
-		UTCNow().Sub(f.offlineTimestamp) >= globalStorageHealthCheckInterval {
+	// Only after initialization has passed (to prevent marking a
```

Comment:
How do we infer that initialization is complete from inside `reInit` method (of `retryStorage`)?

Reply:
Actually this comment doesn't make sense.